### ISSUE TITLE
Join Our E-list Link Opens in a new tab. But the user is not informed of the changes

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#54]
![image](https://cloud.githubusercontent.com/assets/15111528/10883281/fe1a8c64-81c4-11e5-859b-3d1936bd07c5.png)

Tested on Firefox + NVDA combination.

**Comment by HerinHentry** [comment]
@christosptr Can you please check this behaviour with JAWS?

**Comment by christosptr** [comment]
suggested solution 

<a target="_blank" class="btn btn-custom-black" title="Subscribe to TALA Newsletters. Opens New Widnow" href="http://visitor.r20.constantcontact.com/d.jsp?llr=67lifhpab&amp;p=oi&amp;m=1115858153373&amp;sit=lxc8balib&amp;f=b119c4d1-f7ba-41c3-a67b-2afc56cca4fd" aria-lable="Opens New Window">Join E-List</a>

**Comment by mr-bobz** [comment]
Title was already added, adding aria-label="Opens New Window" too. Please recheck now.

**Comment by HerinHentry** [comment]
It works now. But, the Open new window is announced twice as NVDA announces both Title and Aria-Label. This is not an accessibility constraint. So, the issue can be closed.
